src/main/java/com/hollingsworth/arsnouveau/common/items/data/DominionWandData.java

```diff
@@ -21,14 +22,14 @@ public DominionWandData(){
     public static final int NULL_ENTITY = -1;
 
     public static Codec<DominionWandData> CODEC = RecordCodecBuilder.create(instance -> instance.group(
-            BlockPos.CODEC.optionalFieldOf("pos").forGetter(DominionWandData::storedPos),
+            GlobalPos.CODEC.optionalFieldOf("pos").forGetter(DominionWandData::storedPos),
```

Comment:
Does this break existing worlds? If you save a value on the wand as Blockpos, change this to global and then reboot the world, does it still load with that item?

Reply:
good question, will test later

Reply:
not exactly sure what happens, but the world loads fine and the wand is treated as having no stored position. stored entity carries through though.